  uint8 task_mode        #模式 ( 0 自瞄  / 1 大小符)
  uint8 visual_valid     #视觉有效位 (0/1)
  uint8 shoot_valid      #发射有效位 (0/1)
  uint8 reserved         #保留位
  float32 aim_pitch      #欧拉角(度)
  float32 aim_yaw        #欧拉角(度)
  uint8 label